### PR TITLE
Fix (pool transition) 

### PR DESCRIPTION
**Pull request by SorinC6**
Tasks:
- https://emurgo.atlassian.net/browse/YOEXT-1132
- https://emurgo.atlassian.net/browse/YOEXT-1141
- https://emurgo.atlassian.net/browse/YOEXT-1142

### CODE REVIEW

**Comment by zuzunker** [comment]
[YOEXT-1132: [UAT] Pool transition. Incorrect staking dashboard warning banner position, paddings and button sizes](https://emurgo.atlassian.net/browse/YOEXT-1132)

**Comment by zuzunker** [comment]
[YOEXT-1141: [UI] The size of pool transition modal window is different from the design](https://emurgo.atlassian.net/browse/YOEXT-1141)

File: packages/yoroi-extension/app/containers/wallet/dialogs/pool-transition/helpers.js [hunk 1]
```diff
@@ -1,19 +1,22 @@
...
+export const formatTimeSpan = (futureMilliseconds: number, currentMilliseconds: number): string => {
+  let remainingMs = futureMilliseconds - currentMilliseconds;
+
+  if (remainingMs < 0) return 'Date is in the past';
+
+  const days = Math.floor(remainingMs / ms_in_day);
+  remainingMs %= ms_in_day;
+  const hours = Math.floor(remainingMs / ms_in_hour);
+  remainingMs %= ms_in_hour;
+  const minutes = Math.floor(remainingMs / ms_in_min);
```
Comment (zuzunker) [inline packages/yoroi-extension/app/containers/wallet/dialogs/pool-transition/helpers.js]: @SorinC6 , the duration from the `moment` lib can be used nicely here. 

```
const duration = moment.duration(remainingMs);

console.log(duration.days());
console.log(duration.hours());
console.log(duration.minutes());
```

**Review by Nebyt — DISMISSED** [review]

**Review by zuzunker — APPROVED** [review]
Thank you, @SorinC6 !

**Pull request merged**